src/status_im/contexts/wallet/common/utils_test.cljs

```diff
@@ -94,10 +94,10 @@
 
 (deftest test-get-wallet-qr
   (testing "Test get-wallet-qr function"
-    (let [wallet-multichain  {:wallet-type       :wallet-multichain
+    (let [wallet-multichain  {:wallet-type       :multichain
                               :selected-networks [:ethereum :optimism]
                               :address           "x000"}
-          wallet-singlechain {:wallet-type       :wallet-singlechain
+          wallet-singlechain {:wallet-type       :singlechain
```

Comment:
what is singlechain, should this not be legacy?

Reply:
It was there before, and it's testing the functionality of a function. It doesn't make any difference since the function will go into an `else` if the `:wallet-type` is not `multichain`

Reply:
well that's true but it's misleading to someone who views the code later on, probably better to make the underlying `if` use a case/cond and match the same for `legacy` as `default`